src/main/java/edu/wpi/first/gradlerio/deploy/StagedDeployTarget.java

```diff
@@ -89,4 +89,8 @@ private void configureArtifact(Artifact artifact) {
         }
         return depTasks;
     }
+
+    public void setDeployStage(Artifact artifact, DeployStage stage) {
```

Comment:
`this` isn't used here. Why is this function a method here?
This will malfunction if there's more than one target.

Reply:
It's here so you can call it inside the artifacts, which are all inside the targets block. Multiple targets won't affect it at all.

Reply:
```
deploy {
    targets {
        roborio(getTargetTypeClass('RoboRIO')) {
            artifacts {
                frcJava(getArtifactTypeClass('FRCJavaArtifact')) {
                }
                frcStaticFileDeploy(getArtifactTypeClass('FileTreeArtifact')) {
                    setDeployStage(it, 'ProgramKill'); // <<< This works here
                }
            }
        }
    }
}
```